msdb/config/settings/base.py

```diff
@@ -218,7 +233,7 @@
 # Django Admin URL.
 ADMIN_URL = "admin/"
 # https://docs.djangoproject.com/en/dev/ref/settings/#admins
-ADMINS = [("""Kim and Maggie""", "kim-and-maggie@example.com")]
+ADMINS = [("""Kim and Maggie""", "mail@kimdt.tech")]
```

Comment:
can you add two e-mail addresses? fine if not

Reply:
Not possible ):